rules/mfa.md

```diff
@@ -0,0 +1,34 @@
+---
+gallery: true
+short_description: Trigger multifactor authentication when a condition is met
+categories:
+- multifactor
+---
+
+## Multifactor Authentication
+
+This rule is used to trigger multifactor authentication when a condition is met.
+
+```js
+function (user, context, callback) {
+  /*
+  You can trigger MFA conditionally by checking:
+  1. Client ID:
+  context.clientID === 'REPLACE_WITH_YOUR_CLIENT_ID'
+  2. User metadata:
+  user_metadata.use_mfa
```

Comment:
Minor: Maybe show that `user_metadata` is accessed like this: `user.user_metadata.use_mfa`.

Reply:
Fixed, tks!